apps/web/components/v2/eventtype/AvailabilityTab.tsx

```diff
@@ -62,16 +66,54 @@ const format = (date: Date) =>
     new Date(dayjs.utc(date).format("YYYY-MM-DDTHH:mm:ss"))
   );
 
-export const AvailabilityTab = () => {
+export const AvailabilityTab = ({
+  eventTypeTitle,
+  eventTypeId,
+}: {
+  eventTypeTitle: string;
+  eventTypeId: number;
+}) => {
   const { t, i18n } = useLocale();
   const { watch } = useFormContext<FormValues>();
+  const router = useRouter();
 
   const scheduleId = watch("schedule");
   const { isLoading, data: schedule } = trpc.useQuery(["viewer.availability.schedule", { scheduleId }]);
 
+  const createScheduleMutation = trpc.useMutation("viewer.availability.schedule.create", {
+    onSuccess: async ({ schedule }) => {
+      await router.push("/availability/" + schedule.id);
+      showToast(t("schedule_created_successfully", { scheduleName: schedule.name }), "success");
+    },
+    onError: (err) => {
+      if (err instanceof HttpError) {
+        const message = `${err.statusCode}: ${err.message}`;
+        showToast(message, "error");
+      }
+
+      if (err.data?.code === "UNAUTHORIZED") {
+        const message = `${err.data.code}: You are not able to create this event`;
```

Comment:
Did you mean "edit this event" in the error message, or should it say "create availability for this event"?